src/plugins/intel_cpu/src/dnnl_extension_utils.cpp

```diff
@@ -58,8 +60,19 @@ memory::data_type DnnlExtensionUtils::IEPrecisionToDataType(const InferenceEngin
             return memory::data_type::bin;
         case InferenceEngine::Precision::FP16:
             return memory::data_type::f16;
+        case InferenceEngine::Precision::FP64:
+            return memory::data_type::f64;
         case InferenceEngine::Precision::UNSPECIFIED:
             return memory::data_type::undef;
+        // Keep same data_size for unsupported precision
+        case InferenceEngine::Precision::U64:
+        case InferenceEngine::Precision::I64:
+            return memory::data_type::f64;
+        case InferenceEngine::Precision::U32:
+            return memory::data_type::s32;
+        case InferenceEngine::Precision::U16:
+        case InferenceEngine::Precision::I16:
+            return memory::data_type::f16;
```

Comment:
yes, I have updates PR19651

Reply:
For these unsupported precision, if we don't have a solution to handle them, it will cause below test failures:


![image](https://github.com/openvinotoolkit/openvino/assets/31196718/867b80f9-501a-438d-adad-f98df307b707)

The error logs show "CPU plugin doesn't support I16/U16/U32/U64 precision":

![image](https://github.com/openvinotoolkit/openvino/assets/31196718/470e474f-a3ab-45ca-8c82-b57bf95e548b)

Reply:
Then we have to investigate why these tests fail.

Reply:
The test graph is "const->result", the const precision of I16/U16/U32/U64 will hit unsupported precision issue during create DnnlBlockedMemoryDesc.

![image](https://github.com/openvinotoolkit/openvino/assets/31196718/f5d5d758-8f8a-4e34-837e-23dfbbe82114)